svf-llvm/include/SVF-LLVM/SVFIRBuilder.h

```diff
@@ -97,6 +97,7 @@ class SVFIRBuilder: public llvm::InstVisitor<SVFIRBuilder>
     /// GetObject - Return the object node (stack/global/heap/function) according to a LLVM Value
     inline NodeID getObjectNode(const Value* V)
     {
+        V = LLVMUtil::getGlobalRep(V);
         SVFValue* svfVal = LLVMModuleSet::getLLVMModuleSet()->getSVFValue(V);
```

Comment:
I think it is good to put your ` V = LLVMUtil::getGlobalRep(V);` into method `getSVFValue `